### PR TITLE
optimizing build conditions to reduce RHTAP workloads

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
/cc @lance @tommyd450 

based on #79, thanks for your work on this Tommy.

### CODE REVIEW

File: .tekton/trillian-database-1-5-push.yaml [hunk 1]
```diff
@@ -21,6 +20,8 @@ spec:
     value: Dockerfile.database
   - name: git-url
     value: '{{repo_url}}'
+  - name: image-expires-after
+    value: 5d
```
Comment (lance) [inline .tekton/trillian-database-1-5-push.yaml:24]: This should only exist in the pull request pipelines, not the push pipelines. When code is pushed to the production branch, a build is triggered on RHTAP, and any build that was generated from a push event is a release candidate and shouldn't expire.
```suggestion
```

File: .tekton/trillian-logserver-1-5-push.yaml [hunk 2]
```diff
@@ -21,6 +20,8 @@ spec:
     value: Dockerfile.logserver
   - name: git-url
     value: '{{repo_url}}'
+  - name: image-expires-after
+    value: 5d
```
Comment (lance) [inline .tekton/trillian-logserver-1-5-push.yaml:24]: As noted above
```suggestion
```

File: .tekton/trillian-logsigner-1-5-push.yaml [hunk 3]
```diff
@@ -21,6 +20,8 @@ spec:
     value: Dockerfile.logsigner
   - name: git-url
     value: '{{repo_url}}'
+  - name: image-expires-after
+    value: 5d
```
Comment (lance) [inline .tekton/trillian-logsigner-1-5-push.yaml:24]: As noted above
```suggestion
```

File: .tekton/trillian-netcat-1-5-push.yaml [hunk 4]
```diff
@@ -21,6 +20,8 @@ spec:
     value: Dockerfile.netcat
   - name: git-url
     value: '{{repo_url}}'
+  - name: image-expires-after
+    value: 5d
```
Comment (lance) [inline .tekton/trillian-netcat-1-5-push.yaml:24]: As noted above
```suggestion
```

**Review by lance — COMMENTED** [review]
Thanks for picking this up @Gregory-Pereira. Looks good other than the image expiry.

File: .tekton/trillian-database-1-5-push.yaml (hunk 1 above)
Comment (Gregory-Pereira) [reply .tekton/trillian-database-1-5-push.yaml:24]: ack, changing

**Comment by Gregory-Pereira** [comment]
@lance  could you weigh in on why all 8 checks dont run? I would expect ever check for `push` and `pull_request` to run but they are not? Why is this?

**Pull request merged**